Resources/Prototypes/_Sunrise/Entities/Objects/Fun/toys.yml

```
@@ -1115,3 +1115,123 @@
+  components:
+  - type: Sprite
+    sprite: _Sunrise/Objects/Fun/toys.rsi
+    state: plushie-jin
+
+- type: entity
+  id: PlushieAnna
+  parent: BasePlushie
+  name: Plush Anna Solovyova
+  description: She loves her job and isn't shy about expressing her affection for friends, generously showering them with joyful "meows!".
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)